server.js

```diff
@@ -4,23 +4,42 @@ const app = express();
 const bodyParser = require("body-parser");
 app.use(bodyParser.json());
 
-const mockUserData = [
-    { id: 1, name: "Mark" },
-    { id: 2, name: "Jill" },
+const mockUserData = [{
+        id: 1,
+        name: "Mark"
+    },
+    {
+        id: 2,
+        name: "Jill"
+    },
 ];
+//added message and status
+let function_message = '';
+let test_status = false;
+const message = 'successfully got users. Nice!';
+const status = true;
+
 app.get("/users/:id", function (req, res) {
-    res.json({
-        success: true,
-        message: "successfully got users. Nice!",
-        id: req.params.id,
-        user: mockUserData.find((user) => user.id == req.params.id),
-    });
+    if (mockUserData.find((user) => user.id == req.params.id)) {
+        res.json({
+            success: status,
+            message: message,
+            user: mockUserData.find((user) => user.id == req.params.id)
+        });
+    } else {
+        id = req.params.id;
+        function_message = `Sorry user doesn't exist for the id ${id}`;
+        res.json({
+            success: test_status,
+            message: function_message
```

Comment:
Probably doesn't need a variable as it is used in 1 place only

Reply:
Removed it.